fdbclient/MasterProxyInterface.h

```diff
@@ -67,10 +67,11 @@ struct MasterProxyInterface {
 	}
 
 	void initEndpoints() {
-		getConsistentReadVersion.getEndpoint(TaskPriority::ProxyGetConsistentReadVersion);
+		getConsistentReadVersion.getEndpoint(TaskPriority::ReadSocket);
 		getRawCommittedVersion.getEndpoint(TaskPriority::ProxyGetRawCommittedVersion);
-		commit.getEndpoint(TaskPriority::ProxyCommitDispatcher);
+		commit.getEndpoint(TaskPriority::ReadSocket);
 		getStorageServerRejoinInfo.getEndpoint(TaskPriority::ProxyStorageRejoin);
+		getKeyServersLocations.getEndpoint(TaskPriority::ReadSocket); //priority lowered to TaskPriority::DefaultEndpoint on the proxy
```

Comment:
There are some comments related to this endpoint below that I suppose should be updated if we are making this change.